content/en/blog/2024/otel-operator-q-and-a/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: OTel Operator Q&A
+linkTitle: OTel Operator Q&A
+date: 2024-04-24
+author: >-
+  [Adriana Villela](https://github.com/avillela) (ServiceNow),
+
+canonical_url: https://adri-v.medium.com/81d63addbf92?
+cSpell:ignore: automagically mycollector
+---
+
+![Seattle's Mount Rainier rising about the clouds, as seen from an airplane. Photo by Adriana Villela](mount-rainier.jpg)
```

Comment:
Love the picture, though I think there's little point in having it here — on Medium it's a different matter.

Reply:
Let's keep it